distributed/comm/ucx.py

```diff
@@ -39,6 +39,29 @@ def init_once():
     import ucp as _ucp
 
     ucp = _ucp
+    ucx_conf = dask.config.get("ucx")
+
+    enable_infiniband = str(ucx_conf.get("infiniband", None)).lower()
+    enable_nvlink = str(ucx_conf.get("nvlink", None)).lower()
+    tcp_over_ucx = str(ucx_conf.get("tcp_over_ucx", None)).lower()
+    net_devices = ucx_conf.get("net_devices", None)
```

Comment:
For the boolean values can we use bools rather than strings?

Also, we tend to put default values into a yaml file in the project rather than put default values into code like this.  You might add a `ucx:` section to the distributed/distributed.yaml file.

So I think that ideally we would see things like the following:

```python
enable_infiniband = dask.config.get("ucx.infiniband")
```

Or perhaps instead we avoid the intermediate value entirely and just query the configuration when needed ...

```python
if dask.config.get("ucx.infiniband"):
    ...
```

I think that this is still concise, and will make it easier for future reviewers so that they won't have to refer back to what `enable_nvlink` is, and then refer back to what `ucx_conf` is.

Reply:
My concern about bool/string is that if user writes `'False` we can get into trouble without knowing it.

Reply:
The intermediate values serve two purposes:
1) check if any of the configs are set and, if so, configure TLS
2) configure specific hardware nvlink, ib, etc

Reply:
Tried to address much of the comments here in https://github.com/dask/distributed/pull/3515/commits/0755036ee9450a6b794ab6da00c6d6d72a90ec5a

Reply:
> My concern about bool/string is that if user writes 'False we can get into trouble without knowing it.

Right, but that's something that if we handle we should handle centrally, not just for the UCX configuration bits.

> The intermediate values serve two purposes:

To be clear, my only request here is that we instead inline these as `dask.config.get("ucx.nvlink")` rather than have separate variables `ucx_config` and `enable_nvlink`.  I'm only suggesting a cosmetic change.